src/components/ChoiceGroup/ChoiceGroup.tsx

```diff
@@ -0,0 +1,99 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import * as React from 'react';
+import { BaseComponentProps } from '../../util/BaseComponent/props';
+import { ChoiceGroup as FabricChoiceGroup, IChoiceGroupOption } from 'office-ui-fabric-react/lib/ChoiceGroup';
+// prettier-ignore
+import {
+  IChoiceGroupOptionProps,
+} from 'office-ui-fabric-react/lib/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.types';
+import { getStyles, getOptionStyles } from './ChoiceGroup.styles';
+import { join } from '../../util/classNames';
+import Text from '../Text';
+
+export interface ChoiceGroupOption {
+  /**
+   * A required key to uniquely identify the option.
+   */
+  key: string;
+
+  /**
+   * Text for the option.
+   */
+  text: string;
+
+  /**
+   * Label text for the option.
+   */
+  label?: string;
+}
+
+export interface ChoiceGroupProps extends BaseComponentProps {
+  /**
+   * Items to be presented to the user.
+   */
+  options: ChoiceGroupOption[];
+
+  /**
+   * Label text that will be rendered above the choices.
+   */
+  label?: string;
+
+  /**
+   * The selected option.
+   */
+  selectedKey?: ChoiceGroupOption['key'];
+
+  /**
+   * Function to call when user changes the selected item.
+   */
+  onChange(key: ChoiceGroupOption['key']): void;
+}
+
+const getTextWithLabel = (label: string, sublabel: string) => {
+  return () => (
+    <span className="ms-Label">
+      <Text bold={true}>{`${label}:`}</Text>
+      <Text>&nbsp;{sublabel}</Text>
+    </span>
+  );
+};
+
+const mapOption = (option: ChoiceGroupOption): IChoiceGroupOptionProps => {
+  const { text, label } = option;
+  const onRenderLabel = label ? getTextWithLabel(text, label) : undefined;
+
+  return {
+    styles: getOptionStyles,
+    key: option.key,
+    text: text,
+    onRenderLabel,
+  };
+};
+
+/**
+ * The ChoiceGroup component, also known as radio buttons, let users select one
+ * option from two or more choices. Each option is represented by one ChoiceGroup
+ * button; a user can select only one ChoiceGroup in a button group.
+ */
+export default class ChoiceGroup extends React.Component<ChoiceGroupProps> {
+  public render() {
+    const { label, className, options, selectedKey } = this.props;
+    return (
+      <FabricChoiceGroup
+        label={label}
+        className={join(['y-choice-group', className])}
```

Comment:
what does this class do now?

Reply:
Nothing, do you think we should drop it?  I kind of like having it for consumers, but don't feel strongly.

Reply:
i dont see a reason to keep it

Reply:
let's keep it for consistency since all of the YamUI components have a top-level class name to identify them, which also makes it easy to identify the markup in production. we can have a separate discussion if we want to remove these once our components are all using CSS-in-JS